get_linux/index.html

```diff
@@ -0,0 +1,17 @@
+<!DOCTYPE html>
+<head>
+<title>Get QField for Windows</title>
+</head>
+<body>
+<script type="text/javascript">
+fetch('https://api.github.com/repos/opengisch/QField/releases/latest')
+.then(response => {
+    if (response.status !== 200) throw new Error();
+    return response.json();
+})
+.then(json => { window.location.replace('https://github.com/opengisch/QField/releases/download/'+json['tag_name']+'/qfield-'+json['tag_name']+'-linux-x64.AppImage') })
+.catch(error => { window.location.replace('https://github.com/opengisch/QField/releases/latest') });
```

Comment:
```suggestion
const API_LATEST_RELEASE_URL = 'https://api.github.com/repos/opengisch/QField/releases/latest';
const BROWSER_LATEST_RELEASE_URL = 'https://github.com/opengisch/QField/releases/latest';
const queryParams = new Proxy(new URLSearchParams(window.location.search), {
  get: (searchParams, prop) => searchParams.get(prop),
});

if (queryParams.platform) {
  fetch(API_LATEST_RELEASE_URL)
    .then((response) => {
      if (!response.ok) throw new Error();
      return response.json();
    })
    .then(json => {
      const downloadUrl = `https://github.com/opengisch/QField/releases/download/${json['tag_name']}/qfield-${json['tag_name']}-${queryParams['platform']}`;
      if (json['assets'].find(asset => asset["browser_download_url"] === downloadUrl)) {
        return downloadUrl;
      } else {
        return BROWSER_LATEST_RELEASE_URL;
      }
    })
    .then(browserUrl => {
      window.location.replace(browserUrl);
    });
} else {
  window.location.replace(BROWSER_LATEST_RELEASE_URL);
}
```

Reply:
Making it like this means we can add whatever links we want at any point of time and reduce code duplication.

Reply:
I'm fine with that alternative approach. What would be the URL? qfield.org/get_package/?platform={windows,linux} ?

Reply:
Yup, sounds good.